initial Fastlane structures from IzzyOnDroid

IzzySoft opened the pull request:
this PR provides you with a "fastlane starter package". Some notes to sum them up in a place easy to find for you:

* the [IzzyOnDroid Fastlane Documentation](https://gitlab.com/IzzyOnDroid/repo/-/wikis/Fastlane) can guide you with maintaining the structures here
* take special care for limits, e.g. per-release changelogs can have max 500 chars, `full_description.txt` max 4,000 chars, graphics must have specific aspect ratios etc.
* you can have "bigger graphics" here; the IoD updater takes care to resize/optimize them as needed
* for the `full_description.txt` here I've used HTML compressed into a single line, to prevent the fdroid software converting each line break to a `<br>`. The tags used for this are supported by F-Droid.org as well, and to my knowledge even by PlayStore.
* Details on all this in the linked documentation.
* Should you have questions, be welcome to ask :wink:

And now: Enjoy!

trinadhthatakula commented:
is there any restriction for phone screenshots other than being 350px, like max resoltion?

IzzySoft commented:
As outlined, 350px is no limit for you (that's what our importer resizes them to, to save space, bandwidth etc – we always have people with "less capable" devices, on slow networks, and low data plans in mind). There are definitions of aspect ratios somewhere; but in general, what you'd make as screenshots on your device should fit.

We prefer screenshots not to be framed (at least not too much), to deliver as much details as possible without wasting valuable space. Just check how much detail is still available at 350px, and consider small screens :wink:

IzzySoft commented:
Thanks for merging! Activated on our end now, so effective with your next release. Should you add other element types not yet present (e.g. `title.txt`, a featureGraphic, or per-release changelogs) please let us know so we can check and enable them as well. Current settings:

```yaml
  Fastlane: https://github.com/trinadhthatakula/Thor/tree/master/fastlane/metadata
  FastlaneFeatures: shortdesc,fulldesc,icon,screenshots
```

IzzySoft commented:
> I have added title, featureGraphic and change log for v1600

Wonderful! Updated the config on our end to pull them along with releases:

```yaml
  Fastlane: https://github.com/trinadhthatakula/Thor/tree/master/fastlane/metadata
  FastlaneFeatures: title,shortdesc,fulldescMD,changelogs,icon,featureGraphic,screenshots
```

(the `MD` in `fulldescMD` means it's being run through our Markdown processor; I had set it such as people often switch from our one-line-HTML to what we call "Markdown lite", which parses fine as such – and HTML does anyway). Not sure if Playstore handles full HTML, but you will find out.

As there is no new release, but you've added the changelog for the current one… err, wait: NOOO… Please: 1) no HTML in the changelog, and 2) max 500 chars there, see the documentation. That would totally break and look very nasty: it would literally show the HTML code, cut off exactly at position 500. Guess that's not what you wanted :wink:

> If I have any doubts I will conatct you.

Please do! And please let me know once the changelog is fixed (I removed it from the feature list for now, for the reasons mentioned). I can then manually trigger an update from HEAD, so v1600 is covered with changelog.
